src/loaddb/load_db.c

```diff
@@ -1463,7 +1466,18 @@ ldr_check_file_list (std::string & file_name, int &num_files, int &error_code)
       num_files++;
 
       strcpy (schema_object_file->schema_file_name, buffer);
-      read_file_name = buffer;
+
+      last_slash = schema_info_fullpath.find_last_of (PATH_SEPARATOR);
```

Comment:
I have a question, if a user run loaddb in windows like
C> cubrid loaddb --schema-file-list=.C:/testdb/unload/demodb_schema_info testdb
Is it okay?

Reply:
Yes, no problem.

Reply:
I think you have check both '/' and '\' in Windows or use library
```
dirname (path)
_splitpath_s(schema_file_list, drive, _MAX_DRIVE, dir, PATH_MAX, NULL, 0, NULL, 0) 
```